css/webform.css

```diff
@@ -72,3 +72,12 @@ button[name="images_remove_button"] {
     text-decoration: underline;
     color: #eb8d00;
 }
+
+fieldset#edit-location-of-panorama--wrapper .fieldset-wrapper {
```

Comment:
man.. this is just to specific.. what if they need to reuse the same webform component and they give it a different title/name? This will get merged, but you REALLY need to think about this in a more general manner. IDs are no good in Drupal theming since people and ourselves can do site building and it will break things in places we will not even remember.